lib/src/settings.rs

```diff
@@ -265,17 +265,17 @@ impl UserSettings {
 
     /// Looks up string value by `key`.
     pub fn get_string(&self, key: &str) -> Result<String, ConfigError> {
-        self.get(key).and_then(config::Value::into_string)
+        self.get(key)
```

Comment:
Why didn't the code in the second-to-last patch use `self.get_string(key)` instead? That seems closer to what we used before, so it would seem more natural to me.

Reply:
I just wanted to remove `self.config.` references because they'll be replaced with new config API. It's not important because of the last patch, though.